requirements.txt

```diff
@@ -49,11 +49,11 @@ seaborn==0.11.1
 strax==0.15.0
 straxen==0.18.1
 snakeviz==2.1.0
-sphinx==3.5.3
+sphinx==3.5.4
 tables==3.6.1     # pytables, necessary for pandas hdf5 i/o
-tensorflow==2.4.1 # TF2.4.1 should not bring in additional AVX2 requirements (https://github.com/XENONnT/base_environment/issues/433)
+tensorflow==2.5.0-rc1 # TF2.4.1 should not bring in additional AVX2 requirements (https://github.com/XENONnT/base_environment/issues/433)
```

Comment:
Ah I did miss that -- should've looked at the changed files :) 

Is there a reason we need to update numpy right now or can we wait for a full release?

Reply:
I don't consider it super urgent now. I am not aware of any issues with numpy 1.19 but just considered it might have been nice if we could get the more recent version.

Reply:
Ok personally I would prefer to use a slightly outdated numpy version than a pre-release tensorflow. TF has caused some issues in the past, especially on OSG.  

I will leave this open and revert back all my merges from before.

Reply:
Alright, that sounds perfectly sensible. Sorry for not making it not more explicit in this PR.